CHANGES.md

```diff
@@ -28,6 +28,11 @@
 
 ### Removed
 
+- Option --name is removed from all commands. When used with
+  `dune-release distrib`, it was previously effectively ignored. The
+  alternative is to add a `(name <name>)` stanza to
+  `dune-project`. (#324, @lehy)
```

Comment:
This should be #327 instead.
Personally I would replace "The alternative is to add ..." with "Now it is required to add ..."

Reply:
Done.